src/components/PositionedOverlay/PositionedOverlay.tsx

```diff
@@ -195,12 +196,13 @@ export class PositionedOverlay extends React.PureComponent<
           onScrollOut,
           fullWidth,
           fixed,
+          preferInputActivator = true,
         } = this.props;
 
         const textFieldActivator = activator.querySelector('input');
 
         const activatorRect =
```

Comment:
```suggestion
        const activatorRect = getRectForNode(activator);
```

The reason I am making these suggestions is that we won't do an unnecessary `activator.querySelector('input')` if `preferInputActivator` is false,  but also because by letting people specify `preferInputActivator` and then doing this logic, the component might still work but not return what was intended. Does that make sense?

Reply:
yeah, makes sense. We tried to keep this PR as small as possible to make it easier to review and hence didn't want to touch more than needed to introduce our change. We'll be happy to adjust it, though.